libcodechecker/orm_model.py

```diff
@@ -89,25 +87,31 @@ def __init__(self, run_id, checker_name, attribute, value):
         self.checker_name, self.run_id = checker_name, run_id
 
 
+class FileContent(Base):
+    __tablename__ = 'file_content'
```

Comment:
The usual convention is singular class name and plural table name. (Yes I know this is already violated at some places, but no need to violate it further, I have added an action point to my meta-issue #745.)